doxygen/dox/hdf5_2_0.dox

```diff
@@ -91,24 +91,24 @@ Please refer to [Build instructions](https://github.com/HDFGroup/hdf5/blob/hdf5_
 \li chrome –  Download file and then run:  `gzip -cd <distribution>.tar.gz | tar xvf -`
 \li wget –
 <div style="margin-left: 2em;">
-    `wget https://github.com/HDFGroup/hdf5/releases/download/hdf5_1.N.N/<distribution>.tar.gz`<br>
+    `wget https://github.com/HDFGroup/hdf5/releases/download/1.N.N/<distribution>.tar.gz`<br>
```

Comment:
why 1.N.N and not 2.N.N (1.x releases used hdf5_ prefix)?

Reply:
Right